File: 11_oop_abstrakčios klasės/WindowsFormsApp10_gyvunai/WindowsFormsApp10_gyvunai/Zuvis.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace WindowsFormsApp10_gyvunai
{
    internal class Zuvis : Gyvunas
    {
        private int pelekuKiekis;

        public int PelekuKiekis { get => pelekuKiekis; set => pelekuKiekis = value; }

        public Zuvis(String name) : base(name, "zuvis") { }

        public override string ToString()
        {
            return $"Gyvunas: {Type}, vardas: {Name}, peleku kiekis: {PelekuKiekis}";
        }
    }
}

File: 11_oop_abstrakčios klasės/Zuvis.java

public class Zuvis extends Gyvunas{

	private int pelekuKiekis;

	public Zuvis(String name) {
		super(name, "Zuvis");
	}

	@Override
	public String toString() {
		return "" + this.getType() + ". " + "Vardas: " + this.getName() + ", peleku kiekis: " + this.pelekuKiekis;
	}

	public int getPelekuKiekis() {
		return this.pelekuKiekis;
	}

	public void setPelekuKiekis(int pelekuKiekis) {
		this.pelekuKiekis = pelekuKiekis;
	}
	
}
